windows-installer/setup.nsi

```diff
@@ -33,6 +33,9 @@
 ; Directory to install all files into.  This makes uninstall easier and safer.
 !define MAIN_INSTDIR "$INSTDIR\Main"
 
+; Directory for the user to add custom configs.
+!define CUSTOM_CONFIG_DIR "$INSTDIR\config.d"
```

Comment:
Not quite — see my original suggestion with `FLUENTD_CONFIG_DIRECTORY`... That way you ensure that `fluentd.conf` and `config.d` are in the same directory.